packages/ra-ui-materialui/src/layout/Loading.js

```diff
@@ -33,6 +33,9 @@ const useStyles = makeStyles(
     { name: 'RaLoading' }
 );
 
+/**
+ * @type {React.FunctionComponent<{className: string; classes: object; loadingPrimary?: string; loadingSecondary?: string;}>}
```

Comment:
I prefer that you turn that component to TypeScript. I don't want to maintain a lib with 3 different syntax (JS, TS, and JS-with-types)